projects/gloo/pkg/translator/utils.go

```diff
@@ -18,9 +18,35 @@ import (
 	"github.com/solo-io/solo-kit/pkg/api/v1/resources/core"
 )
 
+// ClusterStatNameLabel allows synthetic upstreams to override the
+// stat name at the time of their generation.
+const ClusterStatNameLabel = "internal.solo.io/cluster-stat-name"
+
+// FormatClusterStatName produces a parseable alt_stat_name.
+func FormatClusterStatName(
```

Comment:
do we need to export this? will it be used externally? (besides in the translator test)

Reply:
I may use it in solo-projects. I technically don't have to.